tests/genomics_tests/test_genomic_pipeline.py

```diff
@@ -196,6 +201,19 @@ def setUp(self):
         config.override_setting(config.GENOMIC_GEM_BUCKET_NAME, [_FAKE_GEM_BUCKET])
         config.override_setting(config.GENOMIC_AW1F_SUBFOLDER, [_FAKE_FAILURE_FOLDER])
         config.override_setting(config.RDR_SLACK_WEBHOOKS, self.slack_webhooks)
+        config.override_setting(config.GENOMIC_CVL_SITE_BUCKET_MAP, _FAKE_CVL_SITE_BUCKET_MAP)
+        config.override_setting(config.GENOMIC_GC_SITE_BUCKET_MAP, _FAKE_GC_SITE_BUCKET_MAP)
+        config.override_setting('rdr_bucket_name', [_FAKE_BUCKET])
+        config.override_setting(config.GENOMIC_GC_SITE_PREFIX_MAP, {'rdr': {
+            'cram': 'Wgs_sample_raw_data/CRAMs_CRAIs',
+            'cram.crai': 'Wgs_sample_raw_data/CRAMs_CRAIs',
+            'cram.md5sum': 'Wgs_sample_raw_data/CRAMs_CRAIs',
+            'hard-filtered.vcf.gz': 'Wgs_sample_raw_data/SS_VCF_clinical',
+            'hard-filtered.vcf.gz.md5sum': 'Wgs_sample_raw_data/SS_VCF_clinical',
+            'hard-filtered.vcf.gz.tbi': 'Wgs_sample_raw_data/SS_VCF_clinical',
+            'hard-filtered.gvcf.gz': 'Wgs_sample_raw_data/SS_VCF_research',
+            'hard-filtered.gvcf.gz.md5sum': 'Wgs_sample_raw_data/SS_VCF_research'
+        }})
```

Comment:
Arent these going to be the same for all the GCs?

Can we rename this variable to better encompass what its storing?

Reply:
No, there are differences in capitalization for the file paths depending on the GC so it needs to be set based on the site.

Reply:
So then why are we setting this in the config then? Can you not reuse/utilize/add to the dicts in `genomic_mappings.py`

Reply:
It's very messy to maintain config file mappings like this, config changes should be very simple. We already have this existing type of functionality in the mappings file and would prefer to have them there instead of in the config. If the GC decides for some reason to update their file paths they will let us know.